src/view/observer.js

```diff
@@ -313,10 +313,13 @@ Candy.View.Observer = (function(self, $) {
 	 *
 	 * Parameters:
 	 *   (jQuery.Event) event - jQuery Event object
-	 *   (Object) args - {presetJid}
+	 *   (Object) args - {messageKey, presetJid}
 	 */
 	self.Login = function(event, args) {
-		Candy.View.Pane.Chat.Modal.showLoginForm(null, args.presetJid);
+		args = args || {};
+		args.messageKey = args.messageKey || '';
+		args.presetJid = args.presetJid || null;
```

Comment:
I'm pretty sure you don't need these default assignments. Do them in the method you're calling and go ahead and pass undefined. This is messy :/